data/sql/updates/pending_db_world/update-sai-ssc-trash.sql

```diff
@@ -0,0 +1,120 @@
+(21964, 0, 0, 0, 0, 0, 100, 0, 15750, 15750, 47250, 51550, 0, 0, 11, 38337, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Caribdis - In Combat - Cast Summon Cyclone'),-- fully sniffed
+(21964, 0, 1, 0, 0, 0, 100, 0, 27900, 27900, 6050, 19750, 0, 0, 11, 38335, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Caribdis - In Combat - Cast Water Bolt Volley'),-- fully sniffed
+(21964, 0, 2, 0, 0, 0, 100, 0, 23050, 23050, 24250, 33250, 0, 0, 11, 38358, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Caribdis - In Combat - Cast Tidal Surge'),-- fully sniffed
+(21964, 0, 3, 0, 14, 0, 100, 0, 70000, 200, 15000, 15000, 0, 0, 11, 38330, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Caribdis - Friendly Missing Health - Cast Healing Wave'),-- not sniffed
+(21964, 0, 4, 5, 4, 0, 100, 0, 0, 0, 0, 0, 0, 0, 38, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Caribdis - On Aggro - Set In Combat With Zone'),
+(21964, 0, 5, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 0, 39, 10, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Caribdis - On Aggro - Set In Combat With Zone'),
+(21964, 0, 6, 0, 6, 0, 100, 0, 0, 0, 0, 0, 0, 0, 11, 38451, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Caribdis - On Death - Cast Power of Caribdis'),-- N/A note: missing ability 38449 (on hp%?)
+(21965, 0, 0, 0, 0, 0, 100, 0, 10900, 10900, 10900, 14700, 0, 0, 11, 38234, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Tidalvess - In Combat - Cast Frost Shock'),-- fully sniffed
+(21965, 0, 1, 0, 0, 0, 100, 0, 17000, 17000, 13350, 24250, 0, 0, 11, 38304, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Tidalvess - In Combat - Cast Earthbind Totem'),-- fully sniffed
+(21965, 0, 2, 0, 0, 0, 100, 0, 15000, 15000, 30000, 30000, 0, 0, 11, 38306, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Tidalvess - In Combat - Cast Poison Cleansing Totem'),-- not sniffed
+(21965, 0, 3, 0, 0, 0, 100, 0, 15800, 15800, 24250, 76750, 0, 0, 11, 38236, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Tidalvess - In Combat - Cast Spitfire Totem'),-- fully sniffed
+(21965, 0, 4, 5, 4, 0, 100, 0, 0, 0, 0, 0, 0, 0, 38, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Tidalvess - On Aggro - Set In Combat With Zone'),
+(21965, 0, 5, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 0, 39, 10, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Tidalvess - On Aggro - Set In Combat With Zone'),
+(21965, 0, 6, 0, 6, 0, 100, 0, 0, 0, 0, 0, 0, 0, 11, 38452, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Tidalvess - On Death - Cast Power of Tidalvess'),
+(21966, 0, 0, 0, 0, 0, 100, 0, 0, 1000, 2000, 2000, 0, 0, 11, 38374, 64, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Sharkkis - In Combat - Cast Hurl Trident'),-- not sniffed
+(21966, 0, 1, 0, 0, 0, 100, 0, 53400, 53400, 6050, 22250, 0, 0, 11, 29436, 0, 0, 0, 0, 0, 5, 40, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Sharkkis - In Combat - Cast Leeching Throw'),-- fully sniffed
```

Comment:
Seems like a high initial for such a low repeat. Try checking if there are any conditions for this cast

Reply:
my guess is that it is similar to Nethekurse death coil, aka needs to be at <100% health. I'll check